.github/actions/setup-default-test-properties/test-properties.json

```diff
@@ -14,7 +14,7 @@
   },
   "JavaTestProperties": {
     "SUPPORTED_VERSIONS": ["8", "11", "17", "21"],
-    "FLINK_VERSIONS": ["1.15", "1.16", "1.17", "1.18", "1.19"],
```

Comment:
OK yea we should be able to set a "latest" or "live" flink version here, or create a new section that isn't Java-only since Flink is used in many non-Java situations.

Reply:
yes that could be a follow up